.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Report a problem with concrete
+title: ''
+labels: triage required
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behaviour
+1. first step
+2. second step
+3. etc
+
+**Expected behaviour**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```

Comment:
Would call it  `artifacts` or `logs` or `evidence` (screenshots will suggest to only add screenshot and that is usually painful to work with terminal screenshot)